metroae

```diff
@@ -691,21 +691,17 @@ function pull {
         if [ -w "$SCRIPT_LOG_FILE" ]; then
             debug "${FUNCNAME[0]}: Script log is writable"
             sudo docker pull $METRO_AE_IMAGE:$MAX_CONTAINER_VERSION | tee -a $SCRIPT_LOG_FILE
-            ((status=$?+$PIPESTATUS))
```

Comment:
Why are we removing things from the metroae script for this? Is this for testing?

Reply:
That's weird, I didn't make those changes. I'll fix it

Reply:
Fixed